src/Factory.sol

```diff
@@ -366,6 +371,22 @@ contract Factory is ERC721, Ownable {
         emit Transfer(from, liquidatorAddress, vaultIndex[vault]);
     }
 
+
+    function upgradeVaultVersion(address vault, uint16 version, bytes32[] calldata proofs) external {
+        require(isVault[vault], "FTRY_UVV: Not a vault");
+        require(ownerOf[vaultIndex[vault]] == msg.sender, "FTRY_UVV: You are not the owner");
```

Comment:
Include test

Reply:
already there ser

Reply:
will add more merkle stuff tests though

Reply:
Where is the failing test with "FTRY_UVV: You are not the owner"?